src/views/Logout.vue

```diff
@@ -1,9 +1,15 @@
 <script>
-import { defineComponent, inject } from "@vue/composition-api";
+import { inject } from "@vue/composition-api";
 
-export default defineComponent({
+export default {
```

Comment:
question: why did you remove defineComponent wrapper function?

Reply:
Because i have read this topic: [export default vs. defineComponent](https://stackoverflow.com/questions/66956931/which-way-better-to-create-vue-component-export-default-vs-definecomponent-vs-n). I don't use TS so i removed it.